Modules/_zoneinfo.c

```diff
@@ -693,15 +693,26 @@ zoneinfo_fromutc(PyObject *obj_self, PyObject *dt)
             dt = tmp;
         }
         else {
-            PyObject *replace = PyObject_GetAttrString(tmp, "replace");
-            PyObject *args = PyTuple_New(0);
-            PyObject *kwargs = PyDict_New();
+            PyObject *replace;
+            PyObject *args;
+            PyObject *kwargs;
```

Comment:
Please use intermingled declarations while you're at it :)

Reply:
As far as I understood: you are asking to move these declarations here: https://github.com/python/cpython/blob/1a4a68bd35b026fce1a0bd2bfcd87aecb91b8734/Modules/_zoneinfo.c#L612-L613 ?

If so, I think that it would be less readable, because in all other places declarations and code are mixed.

Reply:
No, I'm asking to move them to where they are used:

- line 700: `PyObject *replace = PyObject_GetAttrString(tmp, "replace");`
- line 705: `PyObject *args = PyTuple_New(0);`
- line 711: `PyObject *kwargs = PyDict_New();`